optaweb-vehicle-routing-backend/src/main/java/org/optaweb/vehiclerouting/domain/VehicleFactory.java

```diff
@@ -25,6 +25,16 @@ private VehicleFactory() {
         throw new AssertionError("Utility class");
     }
 
+    /**
+     * Create vehicle data.
+     * @param name vehicle name
+     * @param capacity vehicle capacity
```

Comment:
@michaltomco Michal, can you help me with this... "vehicle's name/capacity/ID" or just "vehicle name/capacity/ID" without the possessive form? Are both forms correct? Do you prefer one of them?

Reply:
I believe both are correct, even though "vehicle name" sounds more natural to me (and compact).